src/plupload.js

```diff
@@ -2298,6 +2302,7 @@ plupload.File = (function() {
 					// Build multipart request
 					if (options.multipart && canSendMultipart) {
 						xhr.open("post", url, true);
+						xhr.timeout = 90000;
```

Comment:
Can we make it a configurable option with default value (0 - timeout disabled, any other value - timeout enabled) instead of hardcoded value?